src/build_for_init.ps1

```diff
@@ -49,3 +91,8 @@ if ($Build.IsPresent)
     $sb = (Get-Item Function:DoBuild).ScriptBlock
     Invoke-PSPackageProjectBuild -BuildScript $sb
 }
+
+if ( $Test.IsPresent ) {
+    $ps = (Get-Process -id $PID).MainModule.FileName
```

Comment:
We should just call `Invoke-PSPackageProjectTest -Type Functional` and `Invoke-PSPackageProjectTest -Type StaticAnalysis` here.